src/Tree/Entity/Repository/NestedTreeRepository.php

```diff
@@ -172,24 +187,69 @@ public function getPathQueryBuilder($node)
      * Get the Tree path query by given $node
      *
      * @param object $node
+     * @phpstan-param array{includeNode?: bool} $options
+     *
+     * $options['includeNode'] = (bool) Whether to include the node itself (default: true)
```

Comment:
If you need to add a description for the method arguments, please use the conventional PHPDoc `@param` annotation:
```suggestion
     * @param array $options
     *    bool "includeNode": Whether to include the node itself (default: true)
     *
     * @phpstan-param array{includeNode?: bool} $options
```

Reply:
Ok, I wasn't aware there was a convention in documenting these. For example here they are described a bit differently: https://github.com/doctrine-extensions/DoctrineExtensions/blob/c51e05ebbeccdabe9d0f09cc264c1657880a2bfa/src/Tree/RepositoryUtilsInterface.php#L47
Your proposed format looks nice, so I can change my comments like that. Do you mind that there will still be a discrepancy with the above example?

Reply:
I think the only implicit convention is to leverage the PHPDoc annotations when possible. We have a hard normalizing the existing codebase. By now, we haven't a rule for the indentation other than consistency, so feel free to come up the way you think is a better fit.

Reply:
in this case, I'd like to suggest:
```
     * @param array $options
     *     includeNode => (bool) Whether to include the node itself (default: true),
```
I think it's a bit more readable having the param name first, not the type, if you have many params and you're trying to scan through them. WDYT?

Reply:
I have no personal preference here. The only thing I can use as base for the syntax I've suggested is the structure used in the PHPDoc [`@param`](https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/param.html) tag:
```
@param [<Type>] [name] [<description>]
```

Reply:
I updated the phpdocs according to this https://stackoverflow.com/a/51032174/2737626